src/calculation/HTMLBuilder.ts

```diff
@@ -85,25 +87,55 @@ Handlebars.registerHelper('highlightCoverage', (localId, context) => {
   return '';
 });
 
+/**
+ * Sort statements into population, then non-functions, then functions
+ */
+export function sortStatements(measure: fhir4.Measure, groupId: string, statements: StatementResult[]) {
+  const group = measure.group?.find(g => g.id === groupId) || measure.group?.[0];
+  const populationSet = new Set(group?.population?.map(p => p.criteria.expression));
+  function alphaCompare(a: StatementResult, b: StatementResult) {
+    return a.statementName > b.statementName ? 1 : b.statementName > a.statementName ? -1 : 0;
```

Comment:
```suggestion
    return a.statementName <= b.statementName ? -1 : 1;
```

I believe this is the same as what you have written. If they're equivalent, statement `a` comes first, which should be fine.